modules/ROOT/pages/editor-premium-upgrade-promotion.adoc

```diff
@@ -0,0 +1,46 @@
+= Premium upgrade promotion
+:navtitle: {productname} Premium upgrade promotion
+:description: Editor options related to turning the Premium promotion display off
+:keywords: upgrade, promotion, premium, button
+
+== Premum upgrade promotion option
+
+{productname} 6.2 and later includes the `promotion` option. It controls the presentation or otherwise of a Tiny-specific promotion button.
+
+*Type:* `+Boolean+`
+
+*Default value:* `+true+` in Community self-hosted instances; otherwise `+false+`. See xref:#premium-upgrade-promotion-defaults[immediately below] for details
+
+*Possible values:* `+true+`, `+false+`
+
+The *Upgrade* promotion appears in the unused corner of the {productname} menu bar and, consequently, does not appear if the menu bar is disabled.
+
+NOTE: these distribution-specific defaults are not fixed. Someone running {productname} as part of a Premium plan can turn this option on. And someone running the Community distribution of {productname} as a self-hosted instance can turn this option off.
+
+[premium-upgrade-promotion-defaults]
+== Premium upgrade promotion defaults
+
+When the Community distribution of {productname} 6.2 is running as a self-hosted instance, an *Upgrade* promotion button appears in the unused corner of the editor menu bar by default. The button does not appear when Community distributions of {productname} are running on the {cloudname}.
+
+This promotion button redirects to a link:{companyurl}/tinymce-self-hosted-premium-features/[promotion page], showcasing the benefits of the variety of xref:plugins#premium-plugins[Premium Plugins] available for {productname}.
+
+When {productname} 6.2 or later is running as part of a Premium plan, the *Upgrade* promotion button is disabled. And it is disabled when running with a Premium plan whether {productname} is running in the {cloudname} or as a self-hosted instance.
+
+The {productname} Community distribution displays the promotion button when running as a self-hosted instance because the `promotion` option is set to `true` by default. To turn this button off in the Community disribution, set the `promotion` option to `false`.
```

Comment:
```suggestion
When the Community distribution of {productname} 6.2 is running as a self-hosted instance, an *Upgrade* promotion button appears in the unused corner of the editor menu bar by default. The button does not appear when Community distributions of {productname} are running on the {cloudname}. To turn this button off in the Community disribution, set the `promotion` option to `false`.

This promotion button redirects to a link:{companyurl}/tinymce-self-hosted-premium-features/[promotion page], showcasing the benefits of the variety of xref:plugins#premium-plugins[Premium Plugins] available for {productname}.

When {productname} 6.2 or later is running as part of a Premium plan, the *Upgrade* promotion button is disabled in both {cloudname} and self-hosted editors.
```

Reply:
The original is structured as follows:

1. The community distribution has the button.

2. What the button does.

3. Premium plans does not have the button.

4. Why community distribution has the button.

5. Removing the button from the community distribution.

6. NB re the button and Premium plans.

Your suggestion re-structures it thus:

1. The community distribution has the button.

2. Removing the button from the community distribution.

3. What the button does.

4. Premium plans does not have the button.

5. Why community distribution has the button.

6. NB re the button and Premium plans.

FWIW, I understand the impulse to bring *has the button* and *how to remove the button* together.

Nonetheless, I believe the original progressive reveal narrative structure flows better.

Bouncing from *remove the button* to *what the button does* feels like a small narrative jerk, to me.